### PR TITLE
OWLS-90888 added negative AI test: Error path due File Permission issue

### PR DESCRIPTION
**Pull request by marinakog**
Create a file on the source image that cannot be accessed by the oracle user, check error in introspector
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/5711/

### CODE REVIEW

File: integration-tests/src/test/resources/auxiliaryimage/negative/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#   For model-in-image, the following two domain resource attributes can
+#   be a directory in the mount path:
+#     1) 'domain.spec.configuration.model.modelHome'
+#     2) 'domain.spec.configuration.model.wdtInstallHome'
+#   Default '/auxiliary'.
+#
+
+FROM busybox
+ARG AUXILIARY_IMAGE_PATH=/auxiliary
+ARG USER=marina
```
Comment (anpanigr) [inline integration-tests/src/test/resources/auxiliaryimage/negative/Dockerfile:21]: can we avoid personal user info

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java [hunk 2]
```diff
@@ -738,6 +738,106 @@ public void testErrorPathDomainWithFailCustomMountCommand() {
     checkPodReady(adminServerPodName, domainUid, domainNamespace);
   }
 
+  /**
+   * Negative Test to create domain with file in auxiliary image not accessible by oracle user.
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java:742]: Can you describe the usecase with little more detail for easy triaging,

File: integration-tests/src/test/resources/auxiliaryimage/negative/Dockerfile (hunk 1 above)
Comment (marinakog) [reply integration-tests/src/test/resources/auxiliaryimage/negative/Dockerfile]: replaced with different user

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java (hunk 2 above)
Comment (marinakog) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java]: added more description

**Review by rjeberhard — APPROVED** [review]

**Review by anpanigr — APPROVED** [review]
LGTM

**Review by sankarpn — CHANGES_REQUESTED** [review]
few comments

**Pull request merged**